src/index.html

```diff
@@ -6,9 +6,43 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="menu menu-hover">
+        <a href="index.html"class="logo">
+          <img src="images/logo.png" alt="Home"
+          class="logoimg">
+        </a>
+
+        <nav class="nav">
+          <div class="navlist">
+            <li> <a href="#" class="navlink Apple is-active">Apple</a>
+            </li>
+            <li> <a href="#"class="navlink">Samsung</a>
+            </li>
```

Comment:
Add a blank line between the links and need to fix code style
```suggestion
            <li class="nav__item">
              <a href="#"class="nav__link">Samsung</a>
            </li>
```